src/common/helpers/gridstack.ts

```diff
@@ -0,0 +1,76 @@
+export const Config = {
+  handle: 'kyn-widget-drag-handle', // set the drag handle
```

Comment:
It's my two cents that this one may be better using a class name or something else. Since later, let's say Bridge, will be able to import and consume an external widget that is not Shidoka's to the dashboard.

Reply:
At the very least, the default config can be overridden.